app/tests/components/Notification.test.tsx

```diff
@@ -0,0 +1,52 @@
+// External imports
+import React from "react";
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import userEvent from "@testing-library/user-event";
+import { config } from "react-transition-group";
+
+// Internal imports
+import { Notification } from "components/components";
+
+config.disabled = true;
+
+describe("Notification", () => {
+  test("Notification component", () => {
+    render(<Notification>This is a warning</Notification>);
+    expect(screen.getByText("This is a warning")).toBeInTheDocument();
+    expect(screen.getByRole("status")).toBeInTheDocument();
+    expect(screen.queryByRole("alert")).not.toBeInTheDocument();
+  });
+
+  test("Notification component closable", async () => {
+    const user = userEvent.setup();
+    render(
+      <Notification show={true} closable>
+        This is a warning
+      </Notification>
+    );
+    await user.click(screen.queryByRole("button")!);
+  });
```

Comment:
Should there be expectations for this test?

Reply:
Whoops! Thanks for catching that. Looks like we'll need tests for our tests 😉